libraries/java8-buildtime/context/build

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "$MAVEN_MIRROR" ]] || [[ -n "$JAVA_TOOL_OPTIONS" ]]; then
+	mkdir -p ~/.m2
+	cat > ~/.m2/settings.xml << EOF
+<settings>
+EOF
```

Comment:
Does this still work if you put the settings.xml file in the algorithm directory rather than the home directory? Feels awkward overwriting files in the home dir.

Reply:
there is a chance that you overwrite there setting file in their source. Seems like this is a little less dangerous

Reply:
This is unchanged from langpack build, if you have an example of a better design of this build script lemmie know!

Reply:
There are some templating operations that go on here, we could have a jinja replacement at build time but that's an extra step that langrunner doesn't already do. I would argue that the current approach is relatively sane (although I could see a world where we have it named something other than `settings.xml`)

Reply:
closing due to inactivity